docs/topics/class-based-views/intro.txt

```diff
@@ -286,10 +286,30 @@ of the method to be decorated as the keyword argument ``name``::
     class ProtectedView(TemplateView):
         template_name = 'secret.html'
 
+Whenever you have a set of common decorators used redundantly you can define
+an iterable of decorators, i.e. a tuple or a list, and use this instead of invoke
+``method_decorator()`` multiple times::
+
+    decorators = (
+        never_cache,
+        login_required
+    )
+
+    @method_decorator(decorators, name='dispatch')
+    class ProtectedView(TemplateView):
+        template_name = 'secret.html'
+
+Remember that the ``request`` object is managed using a last-in-first-out basis,
```

Comment:
I was doubtful about this, if you see the tests there are two decorator function (their name is not really this) `append_exclamation()` and `append_question()` given this tuple:
```
decorators = (
    append_exclamation, 
    append_question
)
```
On actual order it will append `"!?"` to the string output on reverse order it will append `"?!"`.
The point is ordering them using a "do this and then this and then this..." criterion or mimicking the order in which decorators would usually appear.